pkg/utils/cluster.go

```diff
@@ -2,14 +2,30 @@ package utils
 
 import (
 	"context"
+	"fmt"
+	"os"
 
 	"github.com/golang/glog"
 
+	configv1 "github.com/openshift/api/config/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// default Infrastructure resource name for Openshift
+	infraResourceName = "cluster"
+)
+
 func IsSingleNodeCluster(c client.Client) (bool, error) {
+	if os.Getenv("CLUSTER_TYPE") == ClusterTypeOpenshift {
+		return openshiftSNOClusterStatus(c)
+	}
+	return k8sSNOClusterStatus(c)
```

Comment:
nit: can you change the name also here to not have `SNO`

Reply:
Sure thing, done!